kratos/elements/distance_calculation_flux_based_element.h

```diff
@@ -0,0 +1,359 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//			 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Daniel Diez, Pablo Becker
+//
+
+#if !defined(KRATOS_DISTANCE_CALCULATION_FLUX_BASED_ELEMENT)
+#define  KRATOS_DISTANCE_CALCULATION_FLUX_BASED_ELEMENT
+
+// System includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/variables.h"
+#include "includes/serializer.h"
+#include "utilities/geometry_utilities.h"
```

Comment:
I think this one can be moved to the source file.
```suggestion
```

Reply:
it's used for the simplex geometries

Reply:
I know. I meant moving this include to the source (cpp) file as it is not used in the header.

Reply:
ok